src/ir_Fujitsu.cpp

```diff
@@ -290,8 +296,11 @@ void IRFujitsuAC::buildFromState(const uint16_t length) {
     setCmd(kFujitsuAcCmdStayOn);
   // Currently the only way we know how to tell ARRAH2E & ARRY4 apart is if
   // either the raw Filter or Clean setting is on.
-  if (_model == fujitsu_ac_remote_model_t::ARRAH2E && (_.Filter || _.Clean))
-    setModel(fujitsu_ac_remote_model_t::ARRY4);
+  if (_model == fujitsu_ac_remote_model_t::ARRAH2E && (_.Filter || _.Clean)) {
```

Comment:
&& instead of double if?

Reply:
Thanks. Nice catch. I had it like that because I needed to add some debugging statements. I forgot to clean that up when I removed the debug messages.